Makefile

```diff
@@ -4,9 +4,9 @@ SDK_HOME ?= /opt/vertica/sdk
 SHELL = /bin/bash
 VSQL ?= /opt/vertica/bin/vsql
 LOADER_DEBUG = 0
-TARGET ?= /opt/vertica/packages/odbc-loader/lib
+TARGET ?= ./lib
```

Comment:
Changing the target to ./lib causes the CI tests to fail.   Why is this change needed?   Can we revert this to /opt/vertica/packages/odbc-loader/lib?